app/views/games/_draw_pieces.html.erb

```diff
@@ -0,0 +1,38 @@
+<% @board.each_with_index do |x_item, x_index| %>

```

Comment:
Maybe useful to refactor later using helper methods.. Just ping me during the week so we can talk about it.